marketdata/stream/flow.go

```diff
@@ -127,6 +128,10 @@ var ErrConnectionLimitExceeded = errors.New("connection limit exceeded")
 // ErrInvalidCredentials is returned when invalid credentials have been sent by the user
 var ErrInvalidCredentials = errors.New("invalid credentials")
 
+// ErrInsufficientSubscription is returned when the user does not have proper
+// subscription for the requested feed (e.g. SIP)
+var ErrInsufficientSubscription = errors.New("insufficient subscription")
+
```

Comment:
It is a little bit weird that we define `isErrorIrrecoverable()` in `client.go` while `isErrorRetriable()` here.

Reply:
My guess is that it was originally put there because of the usage.